reporters/BaseReporter.js

```diff
@@ -16,7 +16,7 @@ class BaseReporter {
     /**
      * Called whenever site was crawled (either successfully or not)
      * 
-     * @param {{site: string, failures: number, successes: number, urls: number, data: import("../crawler").CollectResult, crawlTimes: Array<Array<number>>, fatalError: Error, numberOfCrawlers: number, regionCode: string}} data 
+     * @param {{site: string, failures: number, successes: number, urls: number, data: import("../crawler").CollectResult | undefined, crawlTimes: Array<Array<number>>, fatalError: Error, numberOfCrawlers: number, regionCode: string}} data 
```

Comment:
Is this related to the change?

Reply:
Ah, I see. Shouldn't TS complain about that? It seems like it could have figure it out from the code, no?

Reply:
I'm not sure how strict the TS checking is when the types are in jsdoc. 🤷